portality/static/js/edges/admin.applications.edge.js

```diff
@@ -8,8 +8,8 @@ $.extend(true, doaj, {
             if (resultobj.admin) {
                 var journals_url = doaj.adminApplicationsSearchConfig.journalsUrl;
                 if (resultobj.admin.current_journal) {
-                    var fvurl = journals_url + '?source=%7B"query"%3A%7B"query_string"%3A%7B"query"%3A"' + resultobj.admin.current_journal + '"%2C"default_operator"%3A"AND"%7D%7D%2C"from"%3A0%2C"size"%3A10%7D';
-                    result += "<strong>Update Request For</strong>: <a href='" + fvurl + "'>" + resultobj.admin.current_journal + '</a>';
+                    var fvurl = journals_url + '?source=%7B"query"%3A%7B"query_string"%3A%7B"query"%3A"' + edges.escapeHtml(resultobj.admin.current_journal) + '"%2C"default_operator"%3A"AND"%7D%7D%2C"from"%3A0%2C"size"%3A10%7D';
+                    result += "<strong>Update Request For</strong>: <a href='" + fvurl + "'>" + edges.escapeHtml(resultobj.admin.current_journal) + '</a>';
```

Comment:
Note that these fields are system generated, so they shouldn't need escaping, but also that escaping them shouldn't do any harm